packages/kit/src/runtime/client/client.js

```diff
@@ -559,7 +586,14 @@ export function create_client({ target, session, base, trailing_slash }) {
 				});
 			}
 
-			const loaded = await module.load.call(null, load_input);
+			let loaded;
+
+			try {
+				if (import.meta.env.DEV) loading += 1;
+				loaded = await module.load.call(null, load_input);
+			} finally {
+				if (import.meta.env.DEV) loading -= 1;
+			}
```

Comment:
Will Rollup/Terser/whoever optimize out this try/finally block in production? Should we un-DRY this a bit and have two separate dev and prod mode branches? Am I going overboard in trying to reduce bytes? Probably?

Reply:
pushed a change. reducing bytes is always worthwhile